ghost/core/core/server/services/milestones/service.js

```diff
@@ -1,4 +1,7 @@
 const DomainEvents = require('@tryghost/domain-events');
+const logging = require('@tryghost/logging');
+
+const JOB_TIMEOUT = 1000 * 60 * 60 * 24 * (Math.floor(Math.random() * 3)); // 0 - 4 days;
```

Comment:
This is 0-3 days, but the comment says 0-4

Reply:
Ah yeah, will update